### ISSUE TITLE
404 error on http://sdk.ckeditor.com/samples/savetextarea.php

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor-sdk#162]
This file can be reached from `samples/savetextarea.html`.

**Comment by Reinmar** [comment]
```
GET http://sdk.ckeditor.com/samples/highlight/styles/googlecode.css 404 (Not Found)
```

**Comment by Tade0** [comment]
This seems to be a file that was supposed to be responsible for highlighting what's under the "Value" column in the style of google code.

**Comment by Tade0** [comment]
Additional thing: even in the offline version, the form points to sdk.ckeditor.com. This will need to be changed.

**Comment by Reinmar** [comment]
Fixed in #163.
